internal/service/dms/certificate_data_source_test.go

```diff
@@ -0,0 +1,45 @@
+package dms_test
+
+import (
+	"fmt"
+	"testing"
+
+	dms "github.com/aws/aws-sdk-go/service/databasemigrationservice"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-provider-aws/internal/acctest"
+)
+
+func TestAccDMSCertificateDataSource_basic(t *testing.T) {
+	ctx := acctest.Context(t)
+	rName := "tf-test-dms-certificate"
```

Comment:
A randomized resource name will avoid conflicts in case test cleanup fails.
```suggestion
	rName := sdkacctest.RandomWithPrefix(acctest.ResourcePrefix)
```